src/plugins/intel_cpu/thirdparty/ACLConfig.cmake

```diff
@@ -351,7 +353,7 @@ elseif(NOT TARGET arm_compute::arm_compute)
     else()
         set(arm_compute build/${OV_CPU_ARM_TARGET_ARCH}/libarm_compute-static.a)
     endif()
-    set(arm_compute_full_path "${ARM_COMPUTE_SOURCE_DIR}/${arm_compute}")
+    set(arm_compute_full_path "${ARM_COMPUTE_BUILD_DIR}/${OV_CPU_ARM_TARGET_ARCH}/libarm_compute-static.a")
```

Comment:
Don't we need to use `"${ARM_COMPUTE_BUILD_DIR}/${arm_compute}"` here?
If we don't - should we keep `arm_compute` variable then?

Reply:
@alvoron corrected